images/hub/jupyterhub_config.py

```diff
@@ -65,6 +65,7 @@
     ('node_selector', 'node-selector'),
 ):
     set_config_if_not_none(c.KubeSpawner, trait, 'singleuser.' + cfg_key)
+c.KubeSpawner.storage_extra_labels = get_config('singleuser.storage-extra-labels', {})
 
```

Comment:
I deleted another update statement here adding a "storage-kind" label. I squashed the commit after i did a rebase, hmmm... The git history became cleaner but the review got harder. Should I add commits instead?

Reply:
I also deleted a line in the hub's PVC.